### PR TITLE
Update crane to serve manifest lists

### PR DESCRIPTION
**Pull request by ipanova**
closes #2385
https://pulp.plan.io/issues/2385

### CODE REVIEW

File: crane/views/v2.py [hunk 1]
```diff
@@ -64,14 +64,29 @@ def name_redirect(relative_path):
 
     if 'manifests' in path_component:
         schema2_data_json = repository.get_schema2_data_for_repo(name_component)
-        if schema2_data_json:
+        manifest_list_data_json = repository.get_manifest_list_data_for_repo(name_component)
+	manifest_list_amd64_tags_json = repository.get_manifest_list_amd64_for_repo(name_component)
+        if schema2_data_json or manifest_list_data_json:
```
Comment (mhrivnak) [inline crane/views/v2.py]: There is some funky indentation happening from here down.

**Comment by ipanova** [comment]
@mhrivnak for testing purposes, i asked a colleagues to create repo with content which would look like https://pastebin.com/Sd2NvALL
https://hub.docker.com/r/dmage/manifest-list-test/tags/

File: docs/index.rst [hunk 2]
```diff
@@ -267,9 +267,17 @@ Example ``apache.conf`` file:
 Release Notes
 -------------
 
+3.0.0
+
+The 3.0.0 release of Crane adds support for the Docker v2 API manifest lists schema version 2.
+Crane now supports version 4 of the json data file produced by Pulp. The new version enables
+Crane to serve manifest lists. In order to decide to wich manifest type to redirect
```
Comment (mhrivnak) [inline docs/index.rst]: s/wich/which/

**Comment by mhrivnak** [comment]
There also appear to be some test issues I'm sure you will clear up before merging.

**Comment by vrutkovs** [comment]
This seems to break /crane/repositories/v2 page: https://gist.github.com/vrutkovs/3bd41048bd918cd4335ca43c2a4652ba

Otherwise works fine, I'll run several more tests for this

**Pull request merged**